### DISCUSSION TITLE
10/24 지윤환 블로그 생성

### DISCUSSION THREAD

**Discussion by ghost** [Orm-camp/blog-feedback#18] [동글동글 3기 피드백]
https://yoonwhan-ji.github.io/ 

저는 깃허브 페이지로 블로그를 생성하였고
앞으로 배운 내용과 제가 취득한 별도의 지식 등을 올려 많은 타인과 공유하고 싶습니다.

**Comment by Orm-camp** [comment]
앞으로 멋지게 꾸며질 윤환님의 깃허브 페이지가 기대됩니다!
과정 중에 배우신 내용 외에도 어떤 내용들이 실리게 될지 궁금하네요 ㅎㅎ
저희 동글동글 활동에서도 공유해주세요!! 🤗
